Sources/BraintreeThreeDSecure/ThreeDSecurePOSTBody.swift

```diff
@@ -0,0 +1,194 @@
+import Foundation
+
+/// The POST body for `v1/payment_methods/\(urlSafeNonce)/three_d_secure/lookup`
+struct ThreeDSecurePOSTBody: Encodable {
+    
+    let accountType: String?
+    let additionalInfo: AdditionalInfo
+    let amount: String
+    let cardAdd: Bool?
+    let challengeRequested: Bool
+    let customFields: [String: String]?
+    let customer: Customer
+    let dataOnlyRequested: Bool
+    let dfReferenceID: String?
+    let exemptionRequested: Bool
+    let requestedExemptionType: String?
+    let requestedThreeDSecureVersion: String?
+
+    init(request: BTThreeDSecureRequest) {
+        self.requestedExemptionType = request.requestedExemptionType.stringValue
+        self.requestedThreeDSecureVersion = "2"
```

Comment:
Can we hradcode this at the top on like 17? I don't think it's currently optional so can for now have it hardcoded there.